src/scripts/cronjobs/emailBreachAlerts/subscriptionHandler.spec.ts

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Logger } from "winston";
+import { MockProcess } from "../../../test/helpers/mockProcess";
+import { mockMessage, MockSubscription } from "../../../test/helpers/pubsub";
+import { mockLogger } from "../../../test/helpers/mockLogger";
+import { MessageSummary, SubscriptionHandler } from "./subscriptionHandler"; // <-- adjust the path
+import { setTimeout } from "timers/promises";
+import { Message, Subscription } from "@google-cloud/pubsub";
+
+describe("SubscriptionHandler", () => {
+  let logger: Logger;
+  let sub: MockSubscription;
+  let proc: MockProcess;
+  let subCloseSpy: jest.SpyInstance<void, []>;
+
+  beforeEach(() => {
+    logger = mockLogger();
+    subCloseSpy = jest.spyOn(MockSubscription.prototype, "close");
+    sub = new MockSubscription();
+    proc = new MockProcess();
+    jest.clearAllMocks();
```

Comment:
This should theoretically be redundant: https://github.com/mozilla/blurts-server/blob/7521902ee1e28880ff4707714fb72974df660b65/jest.config.cjs#L29


(If it's not, that's something to look into I suppose.)

```suggestion
```